src/Core/tests/Unit/ConfigurationTest.php

```diff
@@ -30,6 +30,14 @@ public function testCreate($config, $env, $expected)
         }
     }
 
+    public function testIsDefault()
```

Comment:
Good. 

With a test, this is official =)

Reply:
good catch. Dones